build.gradle.kts

```
@@ -4,26 +4,33 @@ import java.net.URI
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 }
 
 plugins {
-    kotlin("multiplatform") version "1.4.32" apply false
-    id("com.android.application") version "4.1.3" apply false
-    id("com.android.library") version "4.1.3" apply false
-    id("kotlinx-atomicfu") version "0.14.4" apply false
-    id("org.jmailen.kotlinter") version "3.2.0" apply false
-    id("binary-compatibility-validator") version "0.2.3"
+    kotlin("multiplatform") version "1.5.20" apply false
+    id("com.android.application") version "4.2.0" apply false
+    id("com.android.library") version "4.2.0" apply false
+    id("kotlinx-atomicfu") version "0.16.2" apply false
+    id("org.jmailen.kotlinter") version "3.4.5" apply false
     id("org.jetbrains.dokka") version "1.4.30"
-    id("com.vanniktech.maven.publish") version "0.14.0" apply false
+    id("com.vanniktech.maven.publish") version "0.17.0" apply false
     id("net.mbonnin.one.eight") version "0.1"
+
+    // Breaks with:
+    // A problem occurred configuring project ':registration'.
+    // > org.gradle.api.internal.tasks.DefaultTaskContainer$DuplicateTaskException: Cannot add task 'apiBuild' as a task with that name already exists.
+    //
+    // Disabling until https://github.com/Kotlin/binary-compatibility-validator/issues/38 is fixed.
+    //    id("binary-compatibility-validator") version "0.6.0"
 }
 
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        maven(url = "https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven")
```

nit: for consistency, drop the name for the parameter.

```suggestion
        maven("https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven")
```